launcher/lwjre/lib/calendars.properties

```diff
@@ -21,40 +21,36 @@
 #
```

Comment:
I suspect maybe a search term for files of a particular type dug up stuff under launcher - yeah IMHO everything in there should be ignored (and later on extracted)